radio/src/gui/gui_common.h

```diff
@@ -184,6 +184,19 @@ inline uint8_t MODULE_CHANNELS_ROWS(int moduleIdx)
   }
 }
 
+inline uint8_t IF_ALLOW_RACING_MODE(int moduleIdx)
+{
+  if (!IS_MODULE_ENABLED(moduleIdx)) {
+    return HIDDEN_ROW;
+  }
+  else if (isModulePXX2(moduleIdx)) {
+    if (0 == g_model.moduleData[moduleIdx].channelsCount)
```

Comment:
it is right, but difficult to read, I can't remember if we have a macro for this